frontend/src/HomePage.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+import {
+  Alert,
+  Button,
+  Snackbar,
+  Stack,
+  TextField,
+  Typography,
+} from '@mui/material';
+import useSetPageTitle from './hooks/useSetPageTitle';
+import { pollsApi } from './api';
+import { useLocation } from 'wouter';
+
+export default function HomePage() {
+  const [code, setCode] = useState('');
+  const [snackbarOpen, setSnackbarOpen] = useState(false);
+  const [, setLocation] = useLocation();
+  useSetPageTitle('Home');
+  return (
+    <>
+      <Snackbar
+        open={snackbarOpen}
+        onClose={() => setSnackbarOpen(false)}
+        autoHideDuration={6000}
+        anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
+      >
+        <Alert
+          onClose={() => setSnackbarOpen(false)}
+          severity='error'
+          sx={{ width: '100%' }}
+        >
+          Poll not found.
+        </Alert>
+      </Snackbar>
+      <Stack spacing={2}>
+        <Typography variant='h3'>SolidPolls</Typography>
+        <Typography variant='body1'>Please enter your poll code:</Typography>
+        <TextField
+          label='Poll code'
+          placeholder='12345678'
```

Comment:
Since the label is layouted like a placeholder when the control is not focused, the additional placeholder seems a little bit confusing to me. I think the label is enough to understand what you have to do

Reply:
I removed the placeholder :)